src/main.c

```diff
@@ -8,6 +8,9 @@
 #include "memory.h"
 
 __attribute__((noreturn)) void app_main(void) {
+    init_globals();
+    ui_initial_screen();
+
```

Comment:
I've added the removed initialization code.

Reply:
This was the missing part.